includes/core-changes/globalization/5.0/icu-globalization-api.md

```diff
@@ -4,14 +4,51 @@
 
 #### Change description
 
-Previously, .NET libraries used [National Language Support (NLS)](/windows/win32/intl/national-language-support) APIs for globalization functionality. For example, NLS functions were used to get culture data, such as date and time format patterns, compare strings, and perform string casing in the appropriate culture.
+In previous .NET versions, .NET libraries use [National Language Support (NLS)](/windows/win32/intl/national-language-support) APIs for globalization functionality on Windows. For example, NLS functions were used to compare strings, get culture information, and perform string casing in the appropriate culture.
```

Comment:
This is not true for .NET Framework 1.0, 2.0, 3.0 and 3.5. Maybe we can say Full Framework versions 4.0 and above and .NET Core.